webapp_datastore/settings.py

```diff
@@ -71,11 +71,14 @@
     'administration',
     'api',
     'experiments',
-    'uil.core',
-    'uil.vue',
+    'cdh.core',
+    'cdh.vue',
+
+    'impersonate',
 
     # This has to be last, otherwise it's login/logout pages will take over
     'django.contrib.admin',
+    'django.forms',
 ]
 
```

Comment:
I'm missing ``cdh.core.context_processors.acceptation`` in the template context processers list
(And for that matter, the ``ACCEPTATION`` setting that comes with it)

Reply:
(Altough, I would not say it's required to have it. It's just nice to have a warning on the acceptation server)